code/modules/events/abductor.dm

```diff
@@ -0,0 +1,61 @@
+/datum/round_event_control/abductor
+	name = "Abductors"
+	typepath = /datum/round_event/abductor
+	weight = 5
+	max_occurrences = 1
+
+	earliest_start = 180000 // 30 min
+
+	gamemode_blacklist = list("nuclear","wizard","cult","gang","revolution","abduction","blob")
```

Comment:
cult and gang should totally be included, imo. abductors work best when there's already plenty of chaos to go around.